src/comps/PrivateNavbar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import NavItem from 'react-bootstrap/NavItem';
```

Comment:
you can pull these 3 lines into 1 with destructuring: 

`import { NavItem, Nav, Navbar } from "react-bootstrap";`

Reply:
and I have no idea how to just do that w/ a code review. Will figure it out one day

Reply:
If you do that, the entire react-bootstrap library will be imported. If you do it how I did it, you only import the specific components we want.

It has to do with how react-bootstrap library is structured. Each component in that library is its own export default class within its own file. This means you actually cant use the "{}" brackets at all, in fact - those are only for when there is more than one export on the file you are importing. You theoretically _could_ import all three in one line without the squiggly brackets, but as I said that will end up importing the whole library which is a no-no for build size. Its a massive library.